contents/tutorials/bounce-rate.md

```diff
@@ -1,102 +1,136 @@
 ---
 title: How to calculate bounce rate
-date: 2023-06-27
+date: 2024-12-11
 author:
   - ian-vanagas
+  - bijan-boustani
 showTitle: true
 sidebar: Docs
 tags:
   - insights
   - hogql
   - product analytics
+  - web analytics
 ---
 
-Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+Bounce rate is the percentage of users who leave a page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors. This tutorial shows you how to calculate bounce rate in PostHog.
 
-This tutorial shows you how to calculate bounce rate in PostHog. To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js) and enable "record user sessions"  in [project settings](https://app.posthog.com/project/settings).
+To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js). And ensure the following settings are enabled in your project settings:
```

Comment:
This section feels a bit out of place, since we talk about installation, then jump into defiintion of "How we define bounce rate". I think we should move that bounce rate definition above this
```suggestion
To get started, you need to install PostHog's [web snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js). Then ensure the following settings are enabled in your project settings:
```